experiment/simulation/css/main2.css

```diff
@@ -0,0 +1,360 @@
+.window { z-index:20; }

```

Comment:
Why do we need a main2 file? It looks very similar to main.css

Reply:
Deleted the file